### PR TITLE
Bug 1916188: Dockerfile.deploy: copy sourcecode

### PR DESCRIPTION
**Pull request by vrutkovs**
CI tests don't require the source code to be copied here, but without it 
stage deploy procedure is broken

### CODE REVIEW

File: dist/Dockerfile.deploy/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM quay.io/app-sre/cincinnati:builder AS builder
 
+COPY . .
```
Comment (LalatenduMohanty) [inline dist/Dockerfile.deploy/Dockerfile:3]: We do not need to set the workdir ?

Comment (vrutkovs) [reply dist/Dockerfile.deploy/Dockerfile:3]: Its already set in quay.io image and CI's replacement, no need to override it

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**